src/CLI_Installer/Product.wxs

```diff
@@ -20,41 +20,35 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
     <Property Id="NETCORERUNTIMEFOUNDX86">
       <DirectorySearch Id="NetCoreDirectoryFoundx86" Path="[ProgramFilesFolder]dotnet" >
-        <FileSearch Name="dotnet.exe" MinVersion="3.0"/>
+        <FileSearch Name="dotnet.exe" MinVersion="3.1"/>
       </DirectorySearch>
     </Property>
 
     <Property Id="NETCORERUNTIMEFOUNDX64">
       <DirectorySearch Id="NetCoreDirectoryFoundx64" Path="[ProgramFiles64Folder]dotnet" >
-        <FileSearch Name="dotnet.exe" MinVersion="3.0"/>
+        <FileSearch Name="dotnet.exe" MinVersion="3.1"/>
       </DirectorySearch>
     </Property>
 
     <MediaTemplate EmbedCab="yes" />
 
     <Feature Id="ProductFeature" Title="Axe.Windows CLI" Level="1">
       <ComponentGroupRef Id="ProductComponents" />
-      <ComponentGroupRef Id="NetCoreApp20Components" />
-      <ComponentGroupRef Id="NetCoreApp21Components" />
       <ComponentGroupRef Id="NetCoreApp30Components" />
-      <ComponentGroupRef Id="NetStandard20Components" />
     </Feature>
 
-    <Condition Message="[ProductName] requires .NET Core Runtime 3.0 or newer. Please visit https://dotnet.microsoft.com/download/dotnet-core">
+    <Condition Message="[ProductName] requires .NET Core Runtime 3.1 or newer. Please visit https://dotnet.microsoft.com/download/dotnet-core">
       <![CDATA[Installed OR NETCORERUNTIMEFOUNDX64 OR NETCORERUNTIMEFOUNDX86]]>
     </Condition>
 
     <Directory Id="TARGETDIR" Name="SourceDir" >
       <Directory Id="ProgramFilesFolder" >
         <Directory Id="AxeWindowsCLIFolder" Name ="AxeWindowsCLI" >
-          <Directory Id="INSTALLFOLDER" Name="1.0" >
+          <Directory Id="INSTALLFOLDER" Name="$(var.SemVer)" >
             <Directory Id="RuntimesFolder" Name="runtimes" >
               <Directory Id="WinFolder" Name="win" >
                 <Directory Id="LibFolder" Name="lib" >
-                  <Directory Id="NetCoreApp20Folder" Name="netcoreapp2.0" />
-                  <Directory Id="NetCoreApp21Folder" Name="netcoreapp2.1" />
                   <Directory Id="NetCoreApp30Folder" Name="netcoreapp3.0" />
```

Comment:
No "NetCoreApp31Folder"?

Reply:
There's no `netcoreapp3.1` folder. The `runtimes` tree created by the CLI project includes only `runtimes\unix\lib\netcoreapp3.0` (which we omit since UIA is a Windows-specific technology) and `runtimes\win\lib\netcoreapp3.0`